go/api/commands.go

```diff
@@ -0,0 +1,52 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+// StringCommands defines an interface for the "String Commands" group of Redis commands for standalone and cluster clients.
+//
+// See [redis.io] for details.
+//
+// [redis.io]: https://redis.io/commands/?group=string
+type StringCommands interface {
+	// Set the given key with the given value. The return value is a response from Redis containing the string "OK".
+	//
+	// See [redis.io] for details.
+	//
+	// For example:
+	//
+	//	result := client.Set("key", "value")
+	//
+	// [redis.io]: https://redis.io/commands/set/
+	Set(key string, value string) (string, error)
+
+	// SetWithOptions sets the given key with the given value using the given options. The return value is dependent on the
+	// passed options. If the value is successfully set, "OK" is returned. If value isn't set because of [OnlyIfExists] or
+	// [OnlyIfDoesNotExist] conditions, a zero-value string is returned (""). If [SetOptions#ReturnOldValue] is set, the old
```

Comment:
```suggestion
	// [OnlyIfDoesNotExist] conditions, a zero-value string is returned. If [SetOptions#ReturnOldValue] is set, the old
```
zero-value (aka null) != empty string

Reply:
Modified the comment.